birdCollection.php

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Birds Of Pray Collection</title>
+    <link rel="stylesheet" href="stylesheet.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+SC:wght@700&family=Lora:ital,wght@1,500&family=Playfair+Display:ital@1&family=Shadows+Into+Light&display=swap"
+          rel="stylesheet">
+    <link rel="icon" type="IMG/x-icon" href="IMG/favicon.ico">
+</head>
+<body>
+
+<?php
+require_once 'dbLink.php';
+
+$query = $db->prepare('SELECT `id`, `name`, `color`, `season`, `habitat`, `estimatedUKpopulation`, `ageSpanYear`, `picture` FROM `birds` WHERE `deleted` is NULL;');
+
+$query->execute();
+
+$birdsOfPray= $query->fetchAll();
+?>
+<img class="birdimage" src=IMG/BirdBoxImage.png>
+<div class="title">The Birdbox</div>
+    <a class="addNewBirdLink" href="#addANewBird">add a new bird</a>
+<div class="birdBoxContainer">
+
+    <?php
+    foreach($birdsOfPray as $bird){
+        echo'<div class="birdBox">'.
+            '<div class="subtitle">'. $bird['name']. '<br>'. '<br>'. '</div>'.
+            '<img src="'. $bird["picture"]. '"/>'. '<br>'.
+            '<br>'. 'Colour:'. ' '. $bird['color']. '<br>'. '<br>'.
+            'When is the best time of year to spot them?'. '<br>'. $bird['season']. '<br>'. '<br>'.
+            'What kind of habitat you\'ll find them in:'. '<br>'. $bird['habitat']. '<br>'. '<br>'.
+            'How many are left in the UK?'. ' '. '<br>'. 'There are around'. ' '. $bird['estimatedUKpopulation']. ' '. 'left in the UK.'.  '<br>'. '<br>'.
+            'How long does the'. ' '.  $bird['name']. ' '. 'live for?'.  '<br>'. $bird["ageSpanYear"]. ' '. '<br>'. '<br>'.
+            '<a href="deleteBird.php?id=' . $bird['id']. '"><button class="deleteButton">Delete This Bird</button></a></div>';
+    }
```

Comment:
Don't worry about this now, but a suggestion for next time: try to use padding and margin instead of using `<br>` tags everywhere